Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -15,14 +20,25 @@ public ListViewExtensionsPage()
             this.InitializeComponent();
         }
 
+        public ICommand SampleCommand => new DelegateCommand<PhotoDataItem>(OnExecuteSampleCommand);
+
         public async void OnXamlRendered(FrameworkElement control)
         {
             var sampleListView = control.FindChildByName("SampleListView") as ListView;
 
             if (sampleListView != null)
             {
                 sampleListView.ItemsSource = await new Data.PhotosDataSource().GetItemsAsync();
+                ListViewExtensions.SetCommand(sampleListView, SampleCommand);
```

Comment:
```suggestion
```

We can get rid of this line now as you set the DataContext below so the Binding in the sample just picks it up :)